components/identity-i18n-mgt/org.wso2.carbon.i18n.mgt.ui/src/main/java/org/wso2/carbon/i18n/mgt/ui/I18nEmailMgtConfigServiceClient.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.i18n.mgt.ui;
+
+
+import org.apache.axis2.AxisFault;
+import org.apache.axis2.client.Options;
+import org.apache.axis2.client.ServiceClient;
+import org.apache.axis2.context.ConfigurationContext;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.i18n.mgt.dto.xsd.EmailTemplateDTO;
+import org.wso2.carbon.i18n.mgt.stub.I18NEmailMgtConfigServiceStub;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class I18nEmailMgtConfigServiceClient {
+
+    private static Log log = LogFactory.getLog(I18nEmailMgtConfigServiceClient.class);
+
+    private I18NEmailMgtConfigServiceStub stub;
+
+    public I18nEmailMgtConfigServiceClient(String url,
+                                           ConfigurationContext configContext) throws Exception {
+        try {
+            stub = new I18NEmailMgtConfigServiceStub(configContext, url
+                    + "I18nEmailMgtConfigService");
+        } catch (java.lang.Exception e) {
```

Comment:
package name of "Exception" is redundant. You can simply use "Exception" instead.